lib/internal/bootstrap_node.js

```diff
@@ -257,6 +258,22 @@
     });
   }
 
+  function setupInspectorAPI() {
+    const addCommandLineAPIMethod = process.inspector.addCommandLineAPIMethod;
+    delete process.inspector.addCommandLineAPIMethod;
+    addCommandLineAPIMethod('require', function require(request) {
+      const Module = NativeModule.require('module');
+      const path = NativeModule.require('path');
+      const cwd = tryGetCwd(path);
+
+      const name = '[consoleAPI]';
+      const module = new Module(name);
+      module.filename = path.join(cwd, name);
+      module.paths = Module._nodeModulePaths(cwd);
+      return module.require(request);
+    });
```

Comment:
That was my initial concern, yes, but you mentioned that the .parent property will be different for each require() call.  If that's easy to avoid, I'd do it now.